src/scratchtocatrobat/converter/test_converter.py

```diff
@@ -2077,6 +2078,15 @@ def test_can_convert_set_pen_size_block_with_formula(self):
         assert catr_bricks[0].userVariable.getValue().formulaTree.leftChild.value == "2"
         assert catr_bricks[0].userVariable.getValue().formulaTree.rightChild.value == "32"
 
+    def test_can_convert_set_variable_with_list(self):
+        variable_name = "test_var"
+        project = self.block_converter._catrobat_project
+        catrobat.add_user_variable(project, variable_name, DUMMY_CATR_SPRITE, DUMMY_CATR_SPRITE.getName())
+        scratch_block = ["setVar:to:", "test_var", ["contentsOfList:", self._name_of_test_list]]
+        catr_bricks = self.block_converter._catrobat_bricks_from(scratch_block, DUMMY_CATR_SPRITE)
+        assert isinstance(catr_bricks[0], catbricks.SetVariableBrick)
+        assert catr_bricks[0].getFormulas()[0].getFormulaTree().getElementType() == catElementType.USER_LIST
+
```

Comment:
You should also test the value of the formula here. Otherwise this should be ok.